### ISSUE TITLE
AADTenantDetails: wont work in PS5

### ISSUE THREAD

**Issue by Timsto** [microsoft/Microsoft365DSC#1799]
#### Details of the scenario you tried and the problem that is occurring
because of underlying code "Get-MgOrganization" the output of the DSC Ressource AADTenantDetails is empty. Just in PS5.  PS7 works fine
#### Verbose logs showing the problem
Get-MgOrganization : One or more errors occurred.
At line:1 char:1
+ Get-MgOrganization -Verbose
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-MgOrganization_List1], AggregateException
    + FullyQualifiedErrorId : System.AggregateException,Microsoft.Graph.PowerShell.Cmdlets.GetMgOrganization_List1

#### Suggested solution to the issue
problem with the graph-sdk so nothing that can directly resolved by M365DSC

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
# Export-M365DSCConfiguration -Components @("AADTenantDetails")
```

#### Version of the DSC module that was used ('dev' if using current dev branch)
dev

**Comment by olimeise** [comment]
I have a different behavior with AADTenantDetails.
When I run the following command the DSC resource remains empty.
`Export-M365DSCConfiguration -Components @("AADTenantDetails")`

But when I use this the resource is set.
`Export-M365DSCConfiguration -Components @("AADGroupLifecyclePolicy", "AADTenantDetails")`

Get-MgOrganization -verbose works for me.
Can you reproduce that?

Name                           Value                                                                                                                                                                        
----                           -----                                                                                                                                                                        
PSVersion                      5.1.19041.1320

**Comment by Timsto** [comment]
after updating all modules have the same outcome as you

**Comment by andikrueger** [comment]
Could run a test with the latest version and run the following cmdLet to make sure, you are on the latest version and have the latest dependencies installed:

````powershell
Update-M365DSCDependencies
Uninstall-M365DSCOutdatedDependencies
```

This will install the latest dependencies for the latest version and remove all previous versions of Microsoft365DSC and the older dependencies.

**Comment by olimeise** [comment]
I have installed the latest version of Microsoft365DSC from die PSGallery. After that I ran the two cmdlets.
But now I have some problems with the Microsoft.Graph.Authentication module.
When I changed the dependencies in M365DSC module manually to 1.9.3 for every Graph module, everything works for me again. 

Here are the currently installed versions:

`Microsoft365DSC 1.22.309.1`
`Microsoft.Graph                                 1.9.2`
`Microsoft.Graph.Applications                    1.9.3`
`Microsoft.Graph.Authentication                  1.9.3`
`Microsoft.Graph.DeviceManagement                1.9.3`
`Microsoft.Graph.DeviceManagement.Administration 1.9.3`
`Microsoft.Graph.DeviceManagement.Enrolment      1.9.3`
`Microsoft.Graph.Devices.CorporateManagement     1.9.3`
`Microsoft.Graph.Groups                          1.9.3`
`Microsoft.Graph.Identity.DirectoryManagement    1.9.3`
`Microsoft.Graph.Identity.Governance             1.9.3`
`Microsoft.Graph.Identity.SignIns                1.9.3`
`Microsoft.Graph.Planner                         1.9.3`
`Microsoft.Graph.Teams                           1.9.3`
`Microsoft.Graph.Users                           1.9.3`
`DSCParser                                       1.3.0.3`
`ExchangeOnlineManagement                        2.0.5`
`MicrosoftTeams                                  4.0.0`
`MSCloudLoginAssistant                           1.0.83`
`PnP.PowerShell                                  1.9.0`
`ReverseDSC                                      2.0.0.10`
